### DISCUSSION TITLE
Sleap-export to pb file failing

### DISCUSSION THREAD

**Discussion by KostasTsar** [talmolab/sleap#1074] [Help!]
Hello,

I am sure this is a silly mistake on my end, but I am trying to export my model to a pb file to use in bonsai, but am getting an error.

When running: sleap-export --m folder_name  
I get the error : 'Tensor' object cannot be interpreted as an integer.
My folder contains both the h5 model and training_config.json file.
What am I doing wrong?
Thank you in advance

**Comment by talmo** [comment]
Hi @KostasTsar,

Do you mind sharing the training_config.json file for the model you're trying to export?

Thanks!

Talmo

**Comment by KostasTsar** [comment]
Apologies,

Replying again from here. Yes, thank you. Here is the json file.
Best,
Kostas
________________________________

**Reply by talmo** [reply]
Ah yes, currently we only support single instance and top-down models for exporting to Bonsai. We're planning to support bottom-up at some point but it'll be a bit trickier.

Do you think your data will work with a top-down model -- at least for use with Bonsai?

**Comment by KostasTsar** [comment]
Thank you very much for the swift replies, will try and see.
________________________________
